commands/project/exec.sh

```diff
@@ -21,6 +21,11 @@ function project:exec() {
         fi
     fi
 
+    if ! _serviceRunning ${service}; then
+        _logYellow "Project is not running. Starting .."
```

Comment:
```suggestion
        _logYellow "Project is not running. Starting ..."
```
I personally just prefer 3 dots, that's all :stuck_out_tongue_winking_eye: